src/main/java/de/adesso/kicker/notification/NotificationType.java

```diff
@@ -0,0 +1,7 @@
+package de.adesso.kicker.notification;
+
+public enum NotificationType {
+
+    MESSAGE,
```

Comment:
Yes, it use for the identification of the notification type

Reply:
That isn't what I meant by use case. Do we have a use case for a notification of type `MESSAGE`?

Reply:
yes for example if a request is getting declined. Further it is great for future updates like mesages for endings of seasons tournaments or etc.

Reply:
but it would also be possible to set the attribute on null and use the default case on normal notifications. ut personally i like it more if message is also an enum type

Reply:
We don't need the `messages` attribute as the messages will be generated by the `Services` itself. You can easily read what kind of message it would be by the primary key/foreign key + notificationtype. We also have to focus on our current point of stand. It'll end up in _**feature creep**_ as Tom told us.

Reply:
@ccaylak this i not a string, this is about the enum notificationType

Reply:
That's not the thing. We don't need messages at all as they will be replaced by the messages in `message.properties`. So we don't need the message attribute at all.

Reply:
But in this case we would replace it with an other attribute to identify which message has to be shown and add the usernames involved and etc.

Reply:
I woudn't this **feature creep**, specifying the type of a notification is more like making the system more flexible if we want to add any new type of notification in the future. We should consider removing the enum State MESSAGES as we don't use it right now, but i would not to remove the hole type attribut as we don't need to change our notification system if we want to add new features in the future.

Reply:
The thing is that the text for the messages is currently hard coded into the code. And, as Tom said, we should put them into the `messages.properties`. Because of this the whole attribute becomes obsolete as it's possible to use placeholders in the messages in `messages.properties`.
So this is possbile:
messages.properties
`someMessage=This is a message for {0} and {1}`

And in thymeleaf
`#{someMessage(${some.attribute})}`

Reply:
i am totally agreeing with messages.properies
im just saying that this enum state and the String message attribute are reasonable
i just hardcoded it as long as we hadnt implemented messages.propertis so we'll change that.
@JanSchneider1 the messages enum state is in use in 2 functions

Reply:
You don't seem to realize that we don't need the message attribute because we can use `messages.properties` in the frontend and all the necessary data is already there.

Reply:
Please take a look at my last commit, that implements the properties in the notification context

Reply:
I saw it. We still don't need the attribute.

Reply:
Also it only partially solves the "hardcoded" part. The language is now decided by the one who sends the notification and it's always in that language.

Reply:
Thats a really good point and a reason to change it.

Reply:
I would implement that with another enum "MessageContent" which should be used to identify which text each notification should contain - if you dont have better suggestions.